Sources/Workspace/Workspace.swift

```diff
@@ -535,6 +553,17 @@ extension Workspace {
             diagnostics: diagnostics)
     }
 
+    /// List packages available for update.
+    ///
+    /// - Parameters:
+    ///     - diagnostics: The diagnostics engine that reports errors, warnings
+    ///       and notes.
+    public func dryRun(root: PackageGraphRootInput, diagnostics: DiagnosticsEngine) throws -> [PackageReference: PackageStateChange] {
```

Comment:
We should make the method name more clear, maybe: `updateDependenciesDryRun`?

Reply:
done